frontend/src/app/main/ui/comments.cljs

```diff
@@ -113,6 +113,7 @@
         content  (:content draft)
         pos-x    (* (:x position) zoom)
         pos-y    (* (:y position) zoom)
+        has-content? (and (not= content "") (some? content))
```

Comment:
We have a `str/empty?` function for this.

You need to import it as
`[cuerdas.core :as str]`